libcontainer/init_linux.go

```diff
@@ -541,11 +521,9 @@ func setupUser(config *initConfig) error {
 		return err
 	}
 
-	// if we didn't get HOME already, set it based on the user's HOME
-	if envHome := os.Getenv("HOME"); envHome == "" {
-		if err := os.Setenv("HOME", execUser.Home); err != nil {
```

Comment:
Are there some situations which need this `HOME` env, just like `PATH` env?
We really set the `PATH` env if provided, please see:
https://github.com/opencontainers/runc/blob/97af673ac70a1279c75361dc2aeec44bda965cac/libcontainer/env.go#L49
So, maybe it's safe to set `HOME` env both in here and in `prepareEnv`? I don't know whether this scenario exists or not.

Reply:
I have tested some scenarios to check that, it seems that this will not break anything.
So let's go ahead.